ocs_ci/helpers/helpers.py

```diff
@@ -2790,3 +2791,27 @@ def fetch_used_size(cbp_name, exp_val=None):
             f"matching. Retrying"
         )
     return used_in_gb
+
+
+def get_full_logs_path(cname):
+    """
+    Getting the full path of the logs file for particular test
+
+    Args:
+        cname (obj): the Class object which was run and call this function
```

Comment:
spelling correction : "and called this function"

Reply:
done